### ISSUE TITLE
Error: no such file or directory

### ISSUE THREAD

**Issue by epot** [awrichar/gradle-external-build#5]
The plugin was working fine until today, and now even on previous commits I am having this error on MacOS:

```
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'cmake'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:67)
        ... 4 more
Caused by: java.io.IOException: Cannot run program "cmake" (in directory "xxx/build/external-build/parsing_native/osx_x86-64"): error=2, No such file or directory
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
        ... 6 more
Caused by: java.io.IOException: error=2, No such file or directory
        ... 7 more
```

Did you override an existing plugin release by any chance (using 1.10)?

**Comment by awrichar** [comment]
No, I haven't pushed any plugin release recently (I tagged 1.11 but didn't actually push it out yet). You've been using 1.10? And are you building a cmake project?

**Comment by epot** [comment]
Weird, the release date on mavenrepo says 1.9 and 1.10 were released on the 23th of May 2018, I thought maybe you did a mistake.
Yes I am using 1.10 on a cmake project, and it was working fine until today.

**Comment by awrichar** [comment]
Hmm that is odd - I will double check. Maybe I accidentally triggered a deploy.

**Comment by awrichar** [comment]
The metadata from plugins.gradle.org seems to indicate the current release is still 1.10, last updated on June 2, 2017. Are you pulling from somewhere else?
```
<metadata>
  <groupId>com.cisco.external-build</groupId>
  <artifactId>com.cisco.external-build.gradle.plugin</artifactId>
  <version>1.10</version>
  <versioning>
    <latest>1.10</latest>
    <release>1.10</release>
    <versions>
      <version>1.9</version>
      <version>1.10</version>
    </versions>
    <lastUpdated>20170602145903</lastUpdated>
  </versioning>
</metadata>
```
https://plugins.gradle.org/m2/com/cisco/external-build/com.cisco.external-build.gradle.plugin/maven-metadata.xml

**Comment by epot** [comment]
I looked here, maybe this is not relevant:
https://mvnrepository.com/artifact/com.cisco.external-build/com.cisco.external-build.gradle.plugin

**Comment by epot** [comment]
I tried running the example from `master` as they did not exist on tag `1.10` and here is what I get:

```
$ gradle --version
(...)
------------------------------------------------------------
Gradle 4.7
------------------------------------------------------------

Build time:   2018-04-18 09:09:12 UTC
Revision:     b9a962bf70638332300e7f810689cb2febbd4a6c

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
JVM:          10.0.1 ("Oracle Corporation" 10.0.1+10)
OS:           Mac OS X 10.13.4 x86_64
```

```
$ cmake --version
cmake version 3.11.2
```

```
$ gradle-external-build/examples/cmake > gradle build --stacktrace
> Task :cmake:externalBuildHelloExecutable FAILED

FAILURE: Build failed with an exception.
(...)
Caused by: net.rubygrapefruit.platform.NativeException: Could not start 'cmake'
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:27)
        at net.rubygrapefruit.platform.internal.WrapperProcessLauncher.start(WrapperProcessLauncher.java:36)
        at org.gradle.process.internal.ExecHandleRunner.run(ExecHandleRunner.java:67)
        ... 4 more
Caused by: java.io.IOException: Cannot run program "cmake" (in directory ".../gradle-external-build/examples/cmake/build/external-build/hello/osx_x86-64"): error=2, No such file or directory
        at net.rubygrapefruit.platform.internal.DefaultProcessLauncher.start(DefaultProcessLauncher.java:25)
        ... 6 more
Caused by: java.io.IOException: error=2, No such file or directory
        ... 7 more
```

**Comment by epot** [comment]
Nevermind, I killed all my daemons and it started working again... Not sure what happened there, sorry for the trouble!

**Comment by awrichar** [comment]
No problem; thanks for the update. I published v1.11, so please let me know if you hit any further issues with either the old version or the new one!
